kalite/distributed/static/js/distributed/video/hbtemplates/video-player.handlebars

```diff
@@ -1,5 +1,5 @@
 <div id="video-player"{{#unless content_urls }} class="client-online-only"{{/unless}}>
-    <video class="video-player video-js vjs-default-skin vjs-fullscreen" id="{{ random_id }}" controls preload="auto" width="auto" height="auto" {{#if content_urls.thumbnail }}poster="{{ content_urls.thumbnail }}" {{/if}}data-setup="{}">
```

Comment:
Looks like thumbnail has been removed? Is there a better place we can add it back than here? On slower connections, having that thumbnail show first is helpful. I think we also used it previously for setting the aspect ratio of the player?